presentations/ndcc-overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "National Discovery Cloud for Climate workshop overview"
+date: "2024-03-26"
+format: revealjs
+
+author: "Trey Stafford"
```

Comment:
Don't forget your ORCiD! The decision records have an example of how to format that.

Reply:
Nice suggestion, thanks!